app/helpers/follows_helper.rb

```diff
@@ -52,6 +52,18 @@ def unfollow_drop_id(followable)
     "unfollow-drop-#{entity_full_name(followable)}-#{followable.id}"
   end
 
+  def entity_title(title)
+    entity = title.gsub('::', '/').downcase
```

Comment:
I think you'll love this page http://inflector.me/Budget::Investment :)

This two lines could be just one `t("activerecord.models.#{title.underscore}.other")`